spec/grape/endpoint_spec.rb

```diff
@@ -422,6 +425,7 @@ def app
         get '/declared?first=present'
         expect(last_response.status).to eq(200)
         expect(JSON.parse(last_response.body)['nested']).to be_a(Hash)
+        expect(JSON.parse(last_response.body)['nested']).to eq({})
```

Comment:
what do you think about parsing the response only once?

Reply:
sorry! I'll modify soon.